### PR TITLE
Configure webpack for scss

### LINKED ISSUE #132: feat: set up bulma for .css/.scss customization and theming
More colourful themes have been requested, as the current one is rather drab. To do so, Bulma needs to be set up with Webpack so that changes to .scss files can be built and used. This changes the application from using a web-hosted .css file to using a built one (we might add it to a CDN later when it is properly tweaked and reviewed, as a separate issue). As part of this, documenting how .css is set up will be needed as well.

Subtasks:
- [x] Set up Bulma to be built and used with Webpack as opposed to the current use of a CDN
- [x] Create a theme that is more colourful (but still WCAG 2.0 compliant) for the application
- [ ] Document how to make new themes and modify the current theme

### PR DESCRIPTION
**Pull request by simendag**
# Description
.scss is now responsible for building bulma, including bulma-tooltip. I have included a single theme file which really only replaces the primary colour for now -- but it showcases one way of creating a theme that can be built on later.

In this particular version of the theme, our .png icon is showing its age. We might consider replacing it with an .svg icon (if we can find the one we used to create our favicon and icon) and using a web font to create the branding-text.

# Testing
Try out the various responsive modes and see if anything breaks terribly. The application hasn't really had any logic changed, but give it a try anyway.

## Issues closed
closes #132

### CODE REVIEW

File: src/blue-theme.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@charset "utf-8";
+$democracy-blue: #2e4272;
+$primary: $democracy-blue;
+$tooltip-background-color: $democracy-blue;
+@import "~bulma/bulma";
```
Comment (Log234) [inline src/blue-theme.scss:5]: Codacy Issue found: [Expected empty line before at-rule (at-rule-empty-line-before)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

**Comment by Log234** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity decreasing per file
==============================
+ webpack.config.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655&bid=18146562)

File: src/blue-theme.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+@import "~bulma-tooltip";
```
Comment (Log234) [inline src/blue-theme.scss:6]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss (hunk 1 above)
Comment (simendag) [inline src/blue-theme.scss:5]:
```suggestion

@import "~bulma/bulma";
```

**Comment by Log234** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ webpack.config.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655&bid=18146562)

**Comment by Log234** [comment]
The blue is so dark it appears practically black for elements with thin borders:
image

**Comment by Log234** [comment]
Is it intended that the info boxes remain black?
image
image

**Comment by Log234** [comment]
When I change to 2001 the label of the checkbox for showing all parties does not get enough space:
image
This happens in the full-size version of the site, does not appear to be an issue for the mobile version.

**Comment by Log234** [comment]
The scrollbar for the page is visible, even when the content fit within a single screen:
image

**Comment by Log234** [comment]
Perhaps the menu selection and drop-down selection should have the same colour?
image

**Comment by Log234** [comment]
The drop-down menu does not close when tapping outside the menu on touch devices.
image

**Comment by Log234** [comment]
It does not make sense to show the check-box for whether to show all parties in the Fylkesoversikt, as there are no parties to show:
image

**Comment by Log234** [comment]
The same is true for the Utjevningsmandater view, as all parties in that view have per-definition won at least one seat:
image

**Comment by Log234** [comment]
Not all of these changes are in the scope of this PR, but I propose that we at least create issues for the ones outside the scope.

**Comment by Log234** [comment]
The selection boxes and input boxes in the presentation menu are grossly oversized in the full-size view. All could fit on two rows by scaling them down.
image

File: src/blue-theme.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+@charset "utf-8";
+$democracy-blue: #2e4272;
```
Comment (Log234) [inline src/blue-theme.scss:2]: Maybe we should use a different colour?
There is not enough contrast between the logo colour and the background colour.
image

The blue is so dark it appears practically black for elements with thin borders:
image

**Comment by Log234** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           

Complexity decreasing per file
==============================
+ webpack.config.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655&bid=18146562)

File: src/blue-theme.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+$tooltip-background-color: $democracy-blue;
+
+@import "~bulma/bulma";
+@import "~bulma-tooltip";
+
+[class*=" lavinia-logo"],
+[class^="lavinia-logo"] {
+    stroke: findColorInvert($primary);
+    fill: findColorInvert($primary);
+    width: 90px;
```
Comment (Log234) [inline src/blue-theme.scss:17]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+$primary: $democracy-blue;
+$tooltip-background-color: $democracy-blue;
+
+@import "~bulma/bulma";
+@import "~bulma-tooltip";
+
+[class*=" lavinia-logo"],
+[class^="lavinia-logo"] {
+    stroke: findColorInvert($primary);
+    fill: findColorInvert($primary);
```
Comment (Log234) [inline src/blue-theme.scss:16]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

Comment (Log234) [inline src/blue-theme.scss:16]: Codacy Issue found: [Expected "findColorInvert" to be "findcolorinvert" (function-name-case)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    width: 90px;
+    height: 28px;
```
Comment (Log234) [inline src/blue-theme.scss:18]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+// Update Bulma's global variables
+$primary: $democracy-blue;
+$tooltip-background-color: $democracy-blue;
+
+@import "~bulma/bulma";
+@import "~bulma-tooltip";
+
+[class*=" lavinia-logo"],
+[class^="lavinia-logo"] {
+    stroke: findColorInvert($primary);
```
Comment (Log234) [inline src/blue-theme.scss:15]: Codacy Issue found: [Expected "findColorInvert" to be "findcolorinvert" (function-name-case)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

Comment (Log234) [inline src/blue-theme.scss:15]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss (hunk 5 above)
Comment (oysor) [reply src/blue-theme.scss:16]: Difficult. findColorInvert is a Bulma function.

**Comment by Log234** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity decreasing per file
==============================
+ webpack.config.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655&bid=18146562)

File: src/blue-theme.scss [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+// Update Bulma's global variables
+$primary: $democracy-blue;
+$tooltip-background-color: $democracy-blue;
+
+@import "~bulma/bulma";
+@import "~bulma-tooltip";
+
+[class*=" lavinia-logo"],
+[class^="lavinia-logo"] {
+  stroke: findColorInvert($primary);
```
Comment (Log234) [inline src/blue-theme.scss:15]: Codacy Issue found: [Expected "findColorInvert" to be "findcolorinvert" (function-name-case)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+  fill: findColorInvert($primary);
```
Comment (Log234) [inline src/blue-theme.scss:16]: Codacy Issue found: [Expected "findColorInvert" to be "findcolorinvert" (function-name-case)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

**Comment by Log234** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           

Complexity decreasing per file
==============================
+ webpack.config.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655&bid=18146562)

File: src/blue-theme.scss [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+$primary: $democracy-blue;
+$tooltip-background-color: $democracy-blue;
+
+@import "~bulma/bulma";
+@import "~bulma-tooltip";
+
+[class*=" lavinia-logo"],
+[class^="lavinia-logo"] {
+  letter-spacing:2px;
+  stroke-width:.26458px;
```
Comment (Log234) [inline src/blue-theme.scss:16]: Codacy Issue found: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+  stroke: findColorInvert($primary);
```
Comment (Log234) [inline src/blue-theme.scss:17]: Codacy Issue found: [Expected "findColorInvert" to be "findcolorinvert" (function-name-case)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+// Update Bulma's global variables
+$primary: $democracy-blue;
+$tooltip-background-color: $democracy-blue;
+
+@import "~bulma/bulma";
+@import "~bulma-tooltip";
+
+[class*=" lavinia-logo"],
+[class^="lavinia-logo"] {
+  letter-spacing:2px;
```
Comment (Log234) [inline src/blue-theme.scss:15]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+  stroke-width:.26458px;
+  stroke: findColorInvert($primary);
+  fill: findColorInvert($primary);
```
Comment (Log234) [inline src/blue-theme.scss:18]: Codacy Issue found: [Expected "findColorInvert" to be "findcolorinvert" (function-name-case)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss (hunk 10 above)
Comment (Log234) [inline src/blue-theme.scss:16]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

**Comment by Log234** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           

Complexity decreasing per file
==============================
+ webpack.config.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655&bid=18146562)

File: src/blue-theme.scss (hunk 10 above)
Comment (Log234) [inline src/blue-theme.scss:16]: Codacy Issue found: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

Comment (Log234) [inline src/blue-theme.scss:16]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss (hunk 11 above)
Comment (Log234) [inline src/blue-theme.scss:17]: Codacy Issue found: [Expected "findColorInvert" to be "findcolorinvert" (function-name-case)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss (hunk 13 above)
Comment (Log234) [inline src/blue-theme.scss:18]: Codacy Issue found: [Expected "findColorInvert" to be "findcolorinvert" (function-name-case)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss (hunk 12 above)
Comment (Log234) [inline src/blue-theme.scss:15]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss (hunk 3 above)
Comment (oysor) [reply src/blue-theme.scss:4]: Lavinia-theme

With a white logo it looks ok I think.

Comment (oysor) [reply src/blue-theme.scss:4]: light-blue-theme

Comment (oysor) [reply src/blue-theme.scss:4]: Lavinia-theme

Comment (Log234) [reply src/blue-theme.scss:4]: It looks a lot better with the white logo, the light blue is a bit too bright.
What do you think of this colour? #1340bd

Comment (oysor) [reply src/blue-theme.scss:4]: blue_#1340bd

A bit intense maybe

Comment (Log234) [reply src/blue-theme.scss:4]: Let's just go for the original colour and get this PR through.

**Comment by Log234** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity decreasing per file
==============================
+ webpack.config.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655&bid=18146562)

File: src/blue-theme.scss [hunk 14]
```diff
@@ -0,0 +1,21 @@
...
+$tooltip-background-color: $democracy-blue;
+
+@import "~bulma/bulma";
+@import "~bulma-tooltip";
+
+[class*=" lavinia-logo"],
+[class^="lavinia-logo"] {
+  letter-spacing: 2px;
+  stroke-width: 0.26458px;
+  stroke: findColorInvert($primary);
```
Comment (Log234) [inline src/blue-theme.scss:17]: Codacy Issue found: [Expected "findColorInvert" to be "findcolorinvert" (function-name-case)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+  fill: findColorInvert($primary);
```
Comment (Log234) [inline src/blue-theme.scss:18]: Codacy Issue found: [Expected "findColorInvert" to be "findcolorinvert" (function-name-case)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)

File: src/blue-theme.scss (hunk 3 above)
Comment (oysor) [reply src/blue-theme.scss:4]: Alright

File: src/Layout/Presentation/DistrictOverview/DistrictOverview.tsx [hunk 16]
```diff
@@ -24,20 +24,20 @@ export class DistrictOverview extends React.Component<DistrictOverviewProps, {}>
         const highestVsAverageInPercentage = (1 / highestVotingPower / (1 / averageVotingPower)) * 100;
         const lowestVsAverageInPercentage = (1 / lowestVotingPower / (1 / averageVotingPower)) * 100;
         const mostWeightedDistrict = (
-            <span className="has-text-success">
+            <span className="is-size-4">
```
Comment (Log234) [inline src/Layout/Presentation/DistrictOverview/DistrictOverview.tsx:27]: I propose we simply show normal text, having some words in a larger font feels a bit strange to me. This applies to the other cases where we change from coloured to large text as well.

File: src/blue-theme.scss (hunk 14 above)
Comment (Log234) [reply src/blue-theme.scss:17]: The norm for CSS seems to be using all lower case function names with dash-separated-words. However, it does not seem to be compatible with our setup, so I propose we stick with camelCase function names in CSS and disable the alert.

Comment (oysor) [reply src/blue-theme.scss:17]: Agree

**Comment by Log234** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ webpack.config.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655&bid=18146562)

File: src/Layout/Presentation/DistrictOverview/DistrictOverview.tsx (hunk 16 above)
Comment (oysor) [reply src/Layout/Presentation/DistrictOverview/DistrictOverview.tsx]: Done

**Pull request merged**